Sources/Data/EventQueue/EventInfo.swift

```diff
@@ -30,21 +30,23 @@ public extension EventInfo {
     init(
         forViewingScreenWithName screenName: String,
         screenLabel: String? = nil,
```

Comment:
As of this morning, @chrisrecalis has decided to change this from `screenLabel` to `contentName`.

Reply:
ah yes was just about to push a commit for that